src/collective/upload/tests/test_viewlets.py

```diff
@@ -13,21 +13,19 @@
 import unittest
 
 
-NAME = 'collective.upload.tmpls'
```

Comment:
Before this test file was just for one viewlet, now it is used for two viewlets..

I can add a local variable to use instead.